### ISSUE TITLE
Removed glob option causing problem

### ISSUE THREAD

**Issue by erSitzt** [pteich/consul-kv-search#1]
```
panic: Parse error at position 16:
[-a] [-t] [-p] [-g|-r] [--keys|--values] QUERY
                ^ Undeclared option -g

goroutine 1 [running]:
github.com/jawher/mow%2ecli.(*Cli).Run(0xc0000d7e78, 0xc0000ac020, 0x2, 0x2, 0x0, 0xc0000b6200)
        /Users/peter/go/pkg/mod/github.com/jawher/mow.cli@v1.1.0/cli.go:101 +0x192
main.main()
        /Users/peter/github/consul-kv-search/main.go:82 +0x730
```

maybe because of this still containing `-g`

`app.Spec = "[-a] [-t] [-p] [-g|-r] [--keys|--values] QUERY"`

**Comment by pteich** [comment]
Sorry for the delay. I'll have a look at this!

**Comment by pteich** [comment]
Fixed with v1.2.0 (along with adding a new CLI option 😄 )
